"ci mode app runs them tests on node, nodetap, and browser" test failing

I've cloned the repo, installed the dependencies, and run the tests with `npm test`.  There is one failing test:

```
 162 passing (1s)
  1 failing

  1) ci mode app runs them tests on node, nodetap, and browser:
     Uncaught AssertionError: expected [ Array(6) ] to include 'PhantomJS 1.9'
      at Function.assert.include (/home/duncan/testem/node_modules/chai/lib/chai/interface/assert.js:659:45)
      at EventEmitter.<anonymous> (/home/duncan/testem/tests/ci/ci_tests.js:59:16)
      at Function.spy.notify (/home/duncan/testem/node_modules/ispy/index.js:59:10)
      at EventEmitter.spy [as cleanExit] (/home/duncan/testem/node_modules/ispy/index.js:7:9)
      at EventEmitter.App.exit (/home/duncan/testem/lib/ci/index.js:166:10)
      at Server.g (events.js:180:16)
      at Server.emit (events.js:117:20)
      at net.js:1276:10
      at process._tickCallback (node.js:419:13)
```

--- a/tests/ci/ci_tests.js
+++ b/tests/ci/ci_tests.js
@@ -1,4 +1,4 @@
-var fs = require('fs')    
+var fs = require('fs')
 var App = require('../../lib/ci')
 var TestReporter = require('../../lib/ci/test_reporters/tap_reporter')
 var Config = require('../../lib/config')
@@ -16,7 +16,7 @@ describe('ci mode app', function(){
       done()
     })
   })
-  
+
   it('runs them tests on node, nodetap, and browser', function(done){
     this.timeout(10000)
     var config = new Config('ci', {
@@ -43,27 +43,27 @@ describe('ci mode app', function(){
         assert(helloWorld.every(function(r){
           return r.result.passed
         }), 'hello world should pass')
-        
+
         assert(helloBob.every(function(r){
           return !r.result.passed
         }), 'hello bob should fail')
-        
+
         assert(!nodePlain[0].result.passed, 'node plain should fail')
-        
+
         var launchers = reporter.results.map(function(r){
           return r.launcher
         })
-        
+
         assert.include(launchers, 'Node')
         assert.include(launchers, 'NodePlain')
-        assert.include(launchers, 'PhantomJS 1.9')
+        assert(launchers.some(function(n) { return n.match(/^PhantomJS \d/); }), 'Launchers should include some version of PhantomJS')
 
         assert(reporter.results.length >= 1, 'should have a few launchers') // ball park?
         assert(app.cleanExit.called, 'called process.exit()')
         assert(app.cleanExit.lastCall.args[0], 0)
         done()
       })
-      
+
     })
   })
 
@@ -212,7 +212,7 @@ describe('runHook', function(){
     var app = new App(config)
     stub(app, 'Process').returns(fakeP)
     app.runHook('on_start', function(){
-      assert.equal(app.Process.lastCall.args[0], 
+      assert.equal(app.Process.lastCall.args[0],
         'tunnel dev.app.com:2837 -u http://dev.app.com:2837/')
       done()
     })

